<ui version="4.0" >
 <class>RezzMe</class>
 <widget class="QDialog" name="RezzMe" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RezzMe:// Launcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="font" >
      <font>
       <family>Bitstream Vera Sans</family>
      </font>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <property name="elideMode" >
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="usesScrollButtons" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabSimple" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>502</width>
        <height>266</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Go in-world</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="font" >
          <font>
           <family>Bitstream Vera Sans</family>
          </font>
         </property>
         <property name="text" >
          <string>You seem to have clicked or selected a rezzme:// URI â€” please enter your credentials below:</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAuthenticationName" >
         <property name="font" >
          <font>
           <family>Bitstream Vera Sans</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Grid Authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="labelUser" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="text" >
            <string>&amp;user ID:</string>
           </property>
           <property name="buddy" >
            <cstring>lineEditUser</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineEditUser" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">%(userid)s&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelPassword" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="text" >
            <string>&amp;password:</string>
           </property>
           <property name="buddy" >
            <cstring>lineEditPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="lineEditPassword" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">%(password)s&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QPushButton" name="pushButtonOK" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">click here to start the virtual world client (e.g., the secondlife(tm) client)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>go &amp;in-world</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">click here to cancel&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>&amp;cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Bitstream Vera Sans';">&lt;span style=" font-weight:600;">status:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="indent" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStatus" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font" >
            <font>
             <family>Bitstream Vera Sans</family>
            </font>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="font" >
          <font>
           <family>Bitstream Vera Sans</family>
          </font>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>57</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExpert" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>502</width>
        <height>266</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="labelUri" >
         <property name="toolTip" >
          <string>grid name: %(gridname)s
grid tag: %(gridnick)s</string>
         </property>
         <property name="text" >
          <string>rezzme://cool.opensim.foo.com:9000/CoolPlaza/128/17/23</string>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAuthenticationName2" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Grid authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="1" column="0" >
          <widget class="QLabel" name="labelPassword2" >
           <property name="text" >
            <string>&amp;password:</string>
           </property>
           <property name="buddy" >
            <cstring>lineEditPassword2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="lineEditPassword2" >
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">%(password)s&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="labelUser2" >
           <property name="text" >
            <string>&amp;user ID:</string>
           </property>
           <property name="buddy" >
            <cstring>lineEditUser2</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineEditUser2" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOverride" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>use a &amp;different avatar</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QCheckBox" name="checkBoxBookmark" >
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">if you want to bookmark this place, click here before going in-world&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>&amp;bookmark this place</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>&amp;virtual client to use:</string>
             </property>
             <property name="buddy" >
              <cstring>comboBoxClients</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxClients" >
             <property name="toolTip" >
              <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">select a specific virtual world client here (or add a new one even)&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QPushButton" name="pushButtonOK2" >
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">click here to start the virtual world client (e.g., the secondlife(tm) client)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>go &amp;in-world</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel2" >
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">click here to cancel&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>&amp;cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Bitstream Vera Sans';">&lt;span style=" font-weight:600;">status:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="indent" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStatus2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVersion" >
     <property name="font" >
      <font>
       <family>Bitstream Vera Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:8pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">%(name)s %(version)s&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">author: %(author)s&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">publisher: %(publisher)s&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">license: %(license)s&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">web site: %(url)s&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>vX.Y</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="font" >
      <font>
       <family>Bitstream Vera Sans</family>
      </font>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>labelVersion</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>lineEditUser2</tabstop>
  <tabstop>lineEditPassword2</tabstop>
  <tabstop>checkBoxOverride</tabstop>
  <tabstop>checkBoxBookmark</tabstop>
  <tabstop>comboBoxClients</tabstop>
  <tabstop>pushButtonOK2</tabstop>
  <tabstop>pushButtonCancel2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>RezzMe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>502</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel" >
     <x>415</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel2</sender>
   <signal>clicked()</signal>
   <receiver>RezzMe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>502</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel" >
     <x>358</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
